app/models/move.rb

```diff
@@ -0,0 +1,8 @@
+class Move < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :group_moves, dependent: :destroy
+  has_many :groups, through: :group_moves
+
+  validates :name, presence: true
+  validates :amount, presence: true
```

Comment:
- [ ] Same here, I would recommend validating the type of data too. This avoids any error even if anyone tried to input data from the console.

Reply:
👍🏽